src/argus_ticket_rt.py

```diff
@@ -125,7 +125,7 @@ def create_ticket(cls, serialized_incident: dict):
             authentication_error = "Request Tracker: The authentication details are incorrect. Please check and update the setting 'TICKET_AUTHENTICATION_SECRET'."
             LOG.exception(authentication_error)
             raise TicketSettingsException(authentication_error)
-        except rt_exceptions.NotAllowedError:
+        except (rt_exceptions.NotAllowedError, rt_exceptions.UnexpectedResponseError):
```

Comment:
I would have checked that the status code reported is 403 if that is possible (might be in the exception object.)